api/api/V1/Tag.ts

```diff
@@ -93,15 +96,17 @@ export default function (app: Application, baseUrl: string) {
    * @apiParam {Integer} tagId id of the tag to delete.
    *
    * @apiUse V1ResponseSuccess
-   * @apiuse V1ResponseError
+   * @apiUse V1ResponseError
    *
    */
   app.delete(
     apiUrl,
     extractJwtAuthorizedUser,
     validateFields([idOf(body("tagId"))]),
+
+    // FIXME - Get the recording, if the user has access to it then we're good.
```

Comment:
hmm can anyone delete recording tags at the moment? seems dodgy

Reply:
Nah, it's checked in Tag.deleteFromId, I just want to move that kind of permissions checking to the middleware for consistency.